test/tests/problems/action_custom_fe_problem/tests

```diff
@@ -0,0 +1,39 @@
+[Tests]
+  issues = '#12002'
+  design = ''
+
+  [./no_problem_block]
+    type = 'RunApp'
+    input = 'action_custom_fe_problem_test.i'
+    expect_out = 'Hello, I am your FEProblemBase-derived class and my name is \S+'
+
+    requirement = 'MOOSE shall allow the creation of the Problem block through a User-defined Action.'
+  [../]
+
+  [./with_problem_block_no_type]
+    type = 'RunApp'
+    input = 'action_custom_fe_problem_test.i'
+    expect_out = 'Hello, I am your FEProblemBase-derived class and my name is \S+'
+    cli_args = 'Problem/coord_type=RZ'
```

Comment:
No - they have separate tasks now. CreateProblemAction is responsible for creating and setting parameters in two different tasks.

Reply:
Oh - I see your point... That is something I missed but it's also not an easily solvable problem... This may be a show stopper. I'll have to think more about it.